src/util/colors.ts

```diff
@@ -6,6 +6,11 @@ const COLORS = {
   // background used for items on top of background e.g. buttons
   backgroundPrimary: "#626262",
 
+  backgroundTypeActive: "#002e5f",
+  backgroundTypeInstant: "#1b2e58",
+  backgroundTypeWeapon: "#44005f",
+  backgroundTypeArmor: "#36312f",
```

Comment:
nice! keen to get these in the empty slots too, i might work on that next

Reply:
I was thinking the empty ones would stay using the default bg color, but I'm open to this